src/components/Navbar.js

```diff
@@ -1,16 +1,22 @@
 import { Link } from 'react-router-dom';
+import { BiSolidGroup } from 'react-icons/bi';
 
 const Books = () => (
-  <div className="navbar">
-    <h1>Bookstore CMS</h1>
-    <ul className="">
-      <li className="">
-        <Link to="/">Books</Link>
-      </li>
-      <li>
-        <Link to="/categories">Categories</Link>
-      </li>
-    </ul>
+  <div className="navbarP">
+    <div className="navbar">
+      <h1>Bookstore CMS</h1>
+      <ul className="">
+        <li className="">
+          <Link to="/">Books</Link>
+        </li>
+        <li>
+          <Link to="/categories">Categories</Link>
+        </li>
+      </ul>
+    </div>
+    <div className="Oval">
+      <BiSolidGroup size={40} className="icon" />
```

Comment:
You've made a good start by adding an icon to your project. However, it's important to note that the styling of the `Oval` (I assume this is the icon) doesn't match the design's specifications. To address this:

1. Please adjust the size, padding, background color, and border color of the `Oval` to closely match the design's requirements.

By making these adjustments, you'll ensure that your icon aligns with the design, creating a more cohesive and visually appealing user interface. Keep up the great work! 🌟🖼️